scripts/create_safe_for_agent.py

```diff
@@ -0,0 +1,42 @@
+import secrets
+
+import typer
+from eth_account import Account
+from eth_typing import URI
+from gnosis.eth import EthereumClient
+
+from prediction_market_agent_tooling.tools.gnosis_rpc import GNOSIS_RPC_URL
+from prediction_market_agent_tooling.tools.safe import create_safe
+
+
+def create_safe_for_agent(
+    from_private_key: str = typer.Option(),
+    rpc_url: str | None = None,
+    salt_nonce: int = secrets.randbits(256),
```

Comment:
Is there any reason not to default it to 42 as in the README?

```suggestion
    salt_nonce: int = 42,
```

Reply:
Oh, nevermind, probably because you didn't want to always create the same safe by default.

In that case, beware that `salt_nonce: int = secrets.randbits(256)` is initialised only once, and then it remains the same throughout the whole process. So, if someone decides to use this programmatically for some reason, he will unintentionally get the same safe every time.

Doing `salt_nonce: int | None = None,` and then `salt_nonce = salt_nonce or secrets.randbits(256)` is usually safer.